packages/client/tests/functional/_example/tests.ts

```diff
@@ -50,6 +51,11 @@ testMatrix.setupTestSuite(
       expect(schemaString).toContain('datasource')
       expect(schemaString).toContain('model')
     })
+
+    testIf(suiteConfig.provider !== 'mongodb')('conditional @ts-expect-error', async () => {
+      // @ts-error-if: provider === 'mongodb'
```

Comment:
That what i started with, but unfortuntaly TS will treat it the same way as `ts-expect-error`, does not matter the `-if` at the end. I open to some other suggestions though

Reply:
I have a few ideas:
* @ts-skip-if
* @ts-accept-error-if
* @ts-no-error-if
* @ts-test-if

I prefer `@ts-test-if` because it's 1-to-1 with the `testIf` condition.
So we could copy paste the condition just above, without negation.

Reply:
Agree, will update to `@ts-test-if` then